lmfit/models.py

```diff
@@ -983,6 +984,57 @@ def guess(self, data, x=None, negative=False, **kwargs):
     guess.__doc__ = COMMON_GUESS_DOC
 
 
+class ThermalDistributionModel(Model):
+    r"""Return a thermal distribution function. Variable ``kind`` defines
+    the kind of distribution as below with parameters ``amplitude``
+    (:math:`A`), ``center`` (:math:`x_0`) and ``kT`` (:math:`kT`).
+
+    .. math::
+
+        f(x; A, x_0, kT, T) = \frac{1}{A \exp(\frac{x - x_0}{kT}) + kind}
+
+    As such, these thermal distributions are available:
+
+    - Bose-Einstein distribution via :math:`kind=-1` (the default)
+    - Maxwell-Boltzmann distribution via :math:`kind=0`
+    - Fermi-Dirac distribution via :math:`kind=1`
+
+    see http://hyperphysics.phy-astr.gsu.edu/hbase/quantum/disfcn.html
+
+    Comments:
+
+    - Amplitude and kind are set not to vary by default.
+    - kT should be defined in the same units as x (kB = 8.617e-5 eV/K).
+    - set :math:`kT<0` to implement the energy loss convention common in
+      scattering research.
+    """
```

Comment:
please change:
-  Amplitude to lowercase and add two backticks on both sides as is the convention for variables in docstrings.
- remove ```kind``` from there, that shouldn't be a fitting parameter (see below)
- kT should also have two backticks as well as x

Finally, can you add an empty line before the closing ```"""``` of the docstring?

Reply:
ok